.flake8

```diff
@@ -5,5 +5,11 @@ exclude =
     build,
     dist,
     versioneer.py,
-    doc/conf.py
-max-line-length = 115
+    csxtools/doc/conf.py
+    .ipynb_checkpoints,
```

Comment:
if you use `*.ipynb_checkpoints` then you don't need all 4 of these lines.  just the one - at least this is my recollection and [this seems to confirm](https://learn.microsoft.com/en-us/azure/devops/repos/git/ignore-files?view=azure-devops&tabs=visual-studio-2022)